python-sdk/src/astro/databases/base.py

```diff
@@ -81,23 +82,36 @@ def run_sql(
         self,
         sql_statement: str | ClauseElement,
         parameters: dict | None = None,
+    ):
+        warnings.warn(
+            "`run_sql` is deprecated and will be removed "
+            "Please use  `run_sql_query` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.run_sql_query(sql=sql_statement, parameters=parameters)
+
+    def run_sql_query(
+        self,
+        sql: str | ClauseElement,
+        parameters: dict | None = None,
```

Comment:
```suggestion
        sql: str | ClauseElement,
        parameters: dict | None = None,
```

Reply:
We don't need a new method -- we just need to deprecate a kwarg

Reply:
something like https://github.com/apache/airflow/commit/b220fe60d5302546d85ee15cf6c8eaa5859316d3